docs/evm-sidechain/deploy-a-bridge.md

```diff
@@ -0,0 +1,203 @@
+---
+# Deploy a Bridge
+
+Before deploying a bridge, familiarize yourself with the parameters and terms used:
+
+- **Currency:** The currency used by the bridge. Depending on the token, the bridge has different setups. If the currency is XRP, the door account is called a _Native Bridge Door_; if the currency is a token, the door account is called a _Token Bridge Door_.
+- **Locking Chain:** The chain where the token originally exists. If the locking chain is an XRPL chain, the bridge IOU issuer isn't the bridge door account. If the locking chain is an EVM chain, the ERC20 owner isn't the bridge door account.
+- **Issuing Chain:** The chain the token is bridged to. If the issuing chain is an XRPL chain, the bridged IOU issuer is the bridge door account. If the issuing chain is an EVM chain, the bridged ERC20 owner is the bridge door account and can mint/burn tokens.
+- **Owners:** The witnesses addresses. For each witness, you must have its address on both chains.
+- **Threshold:** The amount of attestations needed for each operation.
+- **Signautre Reward:** The reward distributed to witnesses for providing attestations.
+- **Minimum Create Account:** For native XRP bridges, this parameter sets the minimum amount of XRP that needs to be locked in order to execute a create account operation.
+
+
+## Create a Safe Account
+
+Before deploying a bridge, you have to create a Gnosis Safe. The safe is the multisig account that manages all of the bridge's assets.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/s9J0516Ta64" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+1. Connect a wallet to the [Gnosis Safe management platform](https://safe-evm-poa-sidechain.peersyst.tech/).
+
+![Connect a Wallet](../../images/evm-connect-wallet.png "Connect a Wallet")
+
+2. Click **+ Create New Safe**.
+
+![Create New Safe](../../images/evm-create-safe.png "Create New Safe")
+
+- Give the safe a name to identify it locally.
+- The owners are the bridge witnesses.
+- The threshold is the amount of signatures required to make a transaction on the bridge.
+
+Before submitting, check your information on the review screen.
+
+![Review Safe Info](../../images/evm-safe-owners.png "Review Safe Info")
+
+A prompt will appear, asking to sign the transaction. Carefully review the transaction details before signing it. Once the transaction is submitted, you'll see a success screen.
+
+![Safe Created](../../images/evm-safe-created.png "Safe Created")
+
+
+## Deploy the Bridge Doors
+
+Depending on the bridge currency you want to deploy, there are different steps. Choose the correct bridge type below:
+
+
+### XRP Native Bridge
+
+XRP bridges are a rare case and should only have one bridge per mainchain and sidechain pair. The sidechain specification forces the XRP issuer account to be `rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh`.
```

Comment:
I think it's unnecessarily speculative to call it a rare case. Also, I think you can reduce the amount of sidechain-related jargon in this sentence:

```suggestion
There should should only be at most one XRP native bridge between any two chains. The sidechain specification requires the XRP issuer account to be `rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh`.
```